blog/2024-07-12-airflow-alternatives/index.mdx

```diff
@@ -0,0 +1,192 @@
+---
+slug: airflow-alternatives
+authors: [henricourdent]
+tags: ['Compare']
+description: 'A deep dive on open source alternatives to Apache Airflow: Luigi, Prefect, Dagster, Temporal, Kedro, Windmill, Mage AI, Kestra'
+image: ./compare_airflow.png
+title: 8 Alternatives to Airflow (Strengths and Use Cases)
+---
+
+import UseCasesSvg from '../../static/use_cases.svg';
+import GithubStarTable from '../../src/components/CompareGithub.tsx';
+
+# 8 Alternatives to Airflow - Strengths and Use Cases
+
+[Apache Airflow](https://airflow.apache.org/) is a data orchestration tool. It is used to build, schedule, and monitor workflows. Open source since its inception as a project at [Airbnb in 2014](https://airflow.apache.org/docs/apache-airflow/stable/project.html), Airflow has an active community and is a leading authority in the field of job orchestration.
+
+However, if you have reached this page, it's because Airflow has limitations and modern platforms are better equipped to handle Airflow's tasks, offering a simpler and more complete experience.
+
+This article aims to list 8 open-source alternatives to Airflow: [Luigi](#luigi), [Prefect](#prefect), [Dagster](#dagster), [Temporal](#temporal), [Kedro](#kedro), [Windmill](#windmill), [Mage AI](#mage-ai), [Kestra](#kestra).
+
+These job orchestration tools will be ranked in order of decreasing GitHub stars. This ranking method is far from perfect, but a mathematical criterion is a welcome addition to the subjective world of comparison.
+
+Before we begin, the first thing is to categorize the major aspects we will discuss:
+- **Target users**: for which professions the platform was designed.
```

Comment:
profession is weird, we're not talking about guilds here, use persona or audience